backend/main.py

```
@@ -0,0 +1,58 @@
+    return {"message": "Welcome to Vesta API"}
+
+
+@app.get("/health")
+async def health_check() -> dict:
+    return {"status": "healthy", "message": "Service is running"}
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.12.2)

Possible binding to all interfaces

(S104)